Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1

```diff
@@ -202,27 +210,60 @@ if (!$azureService)
 
 $diagnosticExtensions = Get-DiagnosticsExtensions $StorageAccount $CsCfg
 
+$label = $DeploymentLabel
+
+$appendDateTime = Convert-String $AppendDateTimeToLabel Boolean
+
+if($label -and $appendDateTime)
```

Comment:
We like spaces between our if and opening parentheses.  :-)  there a couple others further down as well.